integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -378,7 +384,8 @@ def test_abs_ansi_overflow(data_type, value):
     assert_gpu_and_cpu_error(
             df_fun=lambda spark: _get_overflow_df(spark, [value], data_type, 'abs(a)').collect(),
             conf=ansi_enabled_conf,
-            error_message='ArithmeticException')
```

Comment:
Do we need/want a similar "We don't check the error message" comment on all of these?

Reply:
Updated!